### ISSUE TITLE
Mac issue - Can't click and drag things into groups

### ISSUE THREAD

**Issue by CONYOfficial** [synfig/synfig#252]
Hi all,

My friend and I are trying to make an animation and found that the newest build of Synfig allows the use of Papagayo. Which would be great if I could actually move our mouth drawings into the specified folders that the .pgo file created! For some reason I have never been able to click and drag groups/objects into one another so that a group encloses something. I have only ever been able to work backwards by importing things I wanted to group right next to each other and then selecting them all, then clicking 'group'. But now I really NEED to put things into these folders, and it won't let me. Clicking and dragging like I see everyone do on the tutorials just doesn't work for some reason. I found a good workaround where I replace the .pgo groups with my own groups, but this is slow and tedious and I would much rather be able to click and drag! 

I use the **Mac** version of Synfig and from what I've been told this is a problem specific to Mac.

**Comment by drsounds** [comment]
Yes I had this problem, I posted this on the previous bug tracker board

**Comment by pgilfernandez** [comment]
I just wanted to ping this issue as it is still present in 1.5.x builds.

Being more specific: drag and drop layers, groups or any item does not work in the Layers Panel. Under macOS.

The workaround consists in relaying in Cut/Copy/Pasting layers, for instance: cut a layer (the one you want to move), then selecting a layer inside a group so that it works as reference for "where to paste" and then pasting; this way the cut layer moves into the group above the selected one and inside the group it belongs.

Cheers

**Comment by marvfash** [comment]
Still can't drag and drop on Mac now.
